containers/Dockerfile.base-spack

```diff
@@ -67,7 +67,7 @@ RUN --mount=type=secret,id=access-nri.pub \
     spack gpg trust /run/secrets/access-nri.pub
 
 # Set up compilers for this base-spack image 
-RUN spack install ${SPACK_ENV_COMPILER_PACKAGE}@${SPACK_ENV_COMPILER_VERSION} target=${SPACK_ENV_TARGET}
+RUN spack install ${SPACK_ENV_COMPILER_PACKAGE}@${SPACK_ENV_COMPILER_VERSION} target=${SPACK_ENV_ARCH}
```

Comment:
If you want to switch to arch, then you'll need to do arch=

Reply:
Noted, updated.